docs/first-model/parameters.rst

```diff
@@ -1,20 +1,66 @@
 Parameter file
 ==============
 
-.. warning::
+Tandem simulation parameters are written in toml script. Here's key parameters and their descriptions:
 
-   This page is under construction.
+- **final_time**: Final simulation time in seconds.
+- **mesh_file**: Mesh file.
+- **lib**: Lua file containing material&frictional paramters.
+- **scenario**: Name of the specific scenario defined in the lua library.
+- **type**: Type of problems. Available options: [poisson | elastic/elasticity].
+- **mode**: Mode of SEAS simulation. Available options: [QuasiDynamic/QD | QuasiDynamicDiscreteGreen/QDGreen | FullyDynamic/FD]
+- **ref_normal**: Define reference normal vector.
+- **up**: Define up direction vector.
+- **boundary_linear**: Assert that boundary is a linear function of time (i.e. boundary(x, t) = f(x) t). Default = false
+- **matrix_free**: Use matrix-free operators. Default = false.
+- **cfl**: CFL tuning parameter (typically >= 1.0). Default = 1.0.
+- **mg_coarse_level**: Polynomial degree of coarsest MG level. Default = 1.
+- **mg_strategy**: MG level selection strategy. Available options: [TwoLevel | Logarithmic | Full]. Default = TwoLevel
+
+When using :code:`mode=QDGreen`, you can use Green's function checkpointing feature by defining following parameters:
```

Comment:
you can use Green's function checkpointing by defining **the** following parameters: